lib/hca/enrollment_system.rb

```diff
@@ -680,12 +680,30 @@ def remove_ctrl_chars!(value)
       end
     end
 
+    def add_dd214(file_body)
+      {
+        'va:document' => {
+          'va:name' => 'DD214',
+          'va:format' => 'PDF',
```

Comment:
Is a `pdf` file type enforced anywhere?  What happens if they upload a Word doc?

Reply:
it's enforced in the dd214 uploader https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/uploaders/hca_dd214_uploader.rb
only images and pdfs are allowed and if they upload an image it gets converted to pdf